packages/app/src/components/Dimensions/DimensionLabel.js

```diff
@@ -54,29 +42,13 @@ export class DimensionLabel extends Component {
     render = () => {
```

Comment:
Just rename the renderLabel fn to render, and get rid of this render fn. No need to break it up like it is currently.